templates/taxi/car_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete car?</h1>
 
```

Comment:
Cancel button will be more user friendly in this case:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.